<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminWidget</class>
 <widget class="QWidget" name="adminWidget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Панель администратора</string>
  </property>
  <widget class="QTabWidget" name="adminSelector">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>721</width>
     <height>481</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="ordersTab">
    <attribute name="title">
     <string>Заказы</string>
    </attribute>
    <widget class="QTableWidget" name="ordersTable">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>711</width>
       <height>431</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Клиент</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Сумма</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Статус оплаты</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Способ получения</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Статус</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Адрес доставки</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Телефон</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Время заказа</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Время готовности</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Состав заказа</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="deleteOrderButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>430</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="getStatsButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>430</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Статистика</string>
     </property>
    </widget>
    <widget class="QLabel" name="orderStatusLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>430</y>
       <width>471</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="foodsTab">
    <attribute name="title">
     <string>Блюда</string>
    </attribute>
    <widget class="QPushButton" name="deleteFoodButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>430</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="foodsTable">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>711</width>
       <height>431</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>511</width>
       <height>0</height>
      </size>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="columnCount">
      <number>4</number>
     </property>
     <column>
      <property name="text">
       <string>Наименование</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Цена</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Свойства</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Ингредиенты</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="editFoodButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>430</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Изменить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addFoodButton">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>430</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Добавить...</string>
     </property>
    </widget>
    <widget class="QLabel" name="foodStatusLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>430</y>
       <width>361</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="productsTab">
    <attribute name="title">
     <string>Продукты</string>
    </attribute>
    <widget class="QTableWidget" name="productsTable">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>711</width>
       <height>431</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>511</width>
       <height>0</height>
      </size>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="columnCount">
      <number>7</number>
     </property>
     <column>
      <property name="text">
       <string>Наименование</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Белки</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Жиры</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Углеводы</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Калории</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Масса</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Цена(за массу)</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="addProductButton">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>430</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Добавить...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteProductButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>430</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="editProductButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>430</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Изменить</string>
     </property>
    </widget>
    <widget class="QLabel" name="productStatusLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>430</y>
       <width>361</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="adminExitButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
